### ISSUE TITLE
Could you supporting OS ventura? plz...

### ISSUE THREAD

**Issue by miranhwang** [slashlos/He3#19]
Hi, Mr.slashlos !!

1. After the update, the temperature is irregular when use He3 in my macbook.
2. And layout gets tangled when use "stage maneger"
3. sometimes, my mouse pointer disappears.

thank you so much.
if you can,,,
could you supporting OS ventura? plz...

**Comment by slashlos** [comment]
Yes it's on the radar; you mean after you upgraded the OS I presume; will do same with the current build to see what happens...

**Comment by miranhwang** [comment]
I hope everything goes well  :D

p.s  Occasionally, the issue of increasing temperature has also been identified.

**Comment by slashlos** [comment]
I'be been running a few week now fine.  I am not familiar with Stage Manager but Helium as a floating browser might be incompatible by its nature. I do not see otherwise performance (heat) impact on a M1pro.  I just installed, so changes not made so perhaps there's something I need to do...

**Comment by slashlos** [comment]
p.s., development is moving to Sonoma 14.2.1 (23C71).

**Comment by slashlos** [comment]
p.s., development is being done on Sequoia 15.7.1 and Tahoe 26.5., in an effort to isolate a loop / hang during to constraints in what appears to be behavior difference for non-storyboard view generations as we do here.

p.p.s a pull request for earlier "[Helium](https://github.com/slashlos/Helium)" (pre 3.0 -  not App Store), I accepted but also exposed this hang, which is **not**  suspected to be related to the request, as I had to migrate to Xcode 26.1.

The pull has bits re: NetFlix I'd like to migrate after the hang is addressed.
